src/WebJobs.Script.WebHost/Security/KeyManagement/SecretManager.cs

```diff
@@ -327,13 +328,7 @@ public async Task<bool> DeleteSecretAsync(string secretName, string keyScope, Sc
                 }
 
                 return secrets;
-            }, secretsFactory).ContinueWith(t =>
-            {
-                if (t.IsFaulted && t.Exception.InnerException is RequestFailedException)
-                {
-                    result = OperationResult.Forbidden;
```

Comment:
Remove OperationResult.Forbidden enum since it's not used anymore

Reply:
done

Reply:
done